pages/place/list/schema/index.jsx

```diff
@@ -25,12 +28,14 @@ module.exports = {
   nacwos: {
     show: true,
     sortable: false,
-    filter: true
+    filter: true,
+    label: <span><Acronym>NACWO</Acronym>s</span>
   },
   nvssqps: {
     show: true,
     sortable: false,
-    filter: true
+    filter: true,
+    label: <span><Acronym>NVSs</Acronym>s / <Acronym>SQP</Acronym>s</span>
```

Comment:
this won't work - wrap it in a fragment

Reply:
ah sorry - just saw the span :D

Reply:
FWIW, unless the `<span>`s are needed for some reason I'd still use a `<Fragment>`